docker/conf/docker-compose-mac.yml.dist

```diff
@@ -24,12 +23,15 @@ services:
             - smtp-server
 
     php-fpm:
-        build: project-base/docker/php-fpm
+        build:
+            context: project-base/docker/php-fpm
+            args:
+                www_data_uid: 1000
```

Comment:
Please change this to 501 for user and 20 for the group in Mac versions, because it is default id.

Reply:
I've changed the defaults for max in fixup commit and solved the GID conflict (20 = dialout)